web_src/js/utils/match.ts

```diff
@@ -41,3 +44,15 @@ export function matchMention(queryText: string): string[] {
 
   return sortAndReduce(results);
 }
+
+export async function matchIssue(owner: string, repo: string, _issueIndex: string, queryText: string): Promise<Issue[]> {
+  const query = queryText.toLowerCase();
+
+  const res = await GET(`/api/v1/repos/${owner}/${repo}/issues?q=${query}&limit=5`);
```

Comment:
Invoking API routers is not allowed, please create a new router under web routers.

Reply:
Moved it to its own endpoint